01-Introduction and Goals.md

```diff
@@ -4,70 +4,24 @@
-
 ## Stakeholders & Expectations
 
-### Wikidata editor
-
-- Add new structured entity data on Wikidata via a desktop web interface.
-- Edit existing structured entity data on Wikidata via a desktop web interface.
-- Programmatically add new structured data to Wikidata.
-- Programmatically edit existing structured data on Wikidata.
-- View existing structured entity data on Wikidata on mobile web.
-- Edit labels, descriptions, aliases on Wikidata via a mobile web interface.
-- Edit terms and sitelinks of entities without using JavaScript in a browser.
-- Have Statements about external identifiers separated from other Statements in the Wikidata UI.
-- Have Statements shown in a useful order.
-- Search on Wikidata for an entity.
-- Ensure the coherence of data modeling in a certain topic.
-- Review edits.
-
-### Wikipedia editor
-
-- Connect pages on different Wikipedias, showing interwiki links on the Articles.
-- Store Wikipedia page connections in a central place.
-- Store infomation relating to article quality in a central place, for use in interwiki links on the Articles.
-- Use data from Wikidata in Wikipedia pages.
-- Have Wikipedia pages automatically update when data on Wikidata changes.
-- Have data updates which cause page changes to be shown in RecentChanges and Watchlists.
-- Update Wikidata’s data used on Wikipedia when it is incorrect or outdated via web interface.
-- See the "unconnected pages" on a Wikipedia to try to create connections.
-- Find wrong or missing data on a Wikipedia article and correct or add it to Wikidata.
-
-#### Small Wikipedia editor
-
-- Have placeholder pages be automatically generated for topics Wikidata contains data for, in order to gain more readers and thus more editors.
-
-### Commons editors
+Stakeholders and expeectations are slowly being sorted into sub component templates.
 
-- Store structured data relevant to files on Commons, on Commons.
-- Describe Commons files using concepts from Wikidata.
+Some stakeholders and expectations may stay at this high level though.
 
-### Data re-user
+### Small Wikipedia editor
 
-- Access Wikidata's content via a web API.
-- Fix and add values on Wikidata.
+- Have placeholder pages be automatically generated for topics Wikidata contains data for, in order to gain more readers and thus more editors.
 
-#### Researcher
+### Wikidata Researcher
 
-- Obtain large parts of the data on Wikidata.
 - Quickly create visualizations based on data from Wikidata without the need to download it.
-- Fix minor modeling and data issues.
-
-### Data donor
-
-- Import (donate) subset of my collection data to Wikidata.
-- Monitor, maintain, and re-export the data donated to Wikidata.
-- Cross-check an external data collection against Wikidata.
 
 ### Non Wikimedia Wikibase Project
```

Comment:
Do you think this whole section could move into the "repo" component docs? Are any 3rd party Wikibase users interested in client functionality?

Reply:
Yes there are many people that use the client functionality (using wikibase data in wikitext pages).
So it probably needs to stay here for now?